main/main.cpp

```diff
@@ -2555,7 +2561,7 @@ bool Main::iteration() {
 		return exit;
 	}
 
-	OS::get_singleton()->add_frame_delay(DisplayServer::get_singleton()->window_can_draw());
+	OS::get_singleton()->add_frame_delay();
```

Comment:
`window_can_draw` depends on the OS/DisplayServer, e.g. window is minimized.
So, frame delay won't be applied when window is minimized, resulting, I think, in much higher CPU usage.

Reply:
Ah, I thought it returned a constant based on the platform.

Do you know if there's a way to make this configurable for the server platform without impacting the behavior on minimized windows?

Reply:
well, more then one, I'm just not sure what's the best approach.
- OS implementations could change `is_in_low_processor_usage_mode` when minimized instead, but that might have side effects on some platforms
- Otherwise `get_low_processor_usage_mode_sleep_usec` could be adjusted when the display is headless.

In general, I don't like the fact that `low_process_mode_sleep_usec` is abused this way, and hard to reconcile the need of different platforms (different window/app lifecycle), display servers (vsync/headless), and the extra constrain of "adding more delay"